index.html

```diff
@@ -0,0 +1,475 @@
+                <div class="services__descriptor--icon icon" aria-hidden="true" data-icon="&#xe001;"></div>
+                <h3>
+                    Brand Identity
+                </h3>
+                <p>Lorem ipsum dolor sit amet, maecenas eget vestibulum justo imperdiet, wisi risus purus augue
+                    vulputate voluptate neque, curabitur dolor libero sodales vitae elit massa.</p>
+            </div>
+            <div class="services__descriptor">
+                <div class="services__descriptor--icon icon" aria-hidden="true" data-icon="&#xe01b;"></div>
+                <h3>
+                    Web Design
+                </h3>
+                <p>Lorem ipsum dolor sit amet, maecenas eget vestibulum justo imperdiet, wisi risus purus augue
+                    vulputate voluptate neque, curabitur dolor libero sodales vitae elit massa. </p>
+            </div>
+            <div class="services__descriptor">
+                <div class="services__descriptor--icon icon" aria-hidden="true" data-icon="&#xe04b;"></div>
+                <h3>
+                    Social Marketing
+                </h3>
+                <p>Lorem ipsum dolor sit amet, maecenas eget vestibulum justo imperdiet, wisi risus purus augue
+                    vulputate voluptate neque, curabitur dolor libero sodales vitae elit massa. </p>
+            </div>
+            <div class="services__descriptor">
+                <div class="services__descriptor--icon icon" aria-hidden="true" data-icon="&#xe027;"></div>
+                <h3>
+                    CMS &amp; eCommerce
+                </h3>
+                <p>Lorem ipsum dolor sit amet, maecenas eget vestibulum justo imperdiet, wisi risus purus augue
+                    vulputate voluptate neque, curabitur dolor libero sodales vitae elit massa. </p>
+            </div>
+        </div>
+    </section>
+    <section id="about">
+        <div class="about__text">
+            <h1 data-section_id="about">Genius Studio</h1>
+            <p>
+                Lorem ipsum dolor sit amet, maecenas eget vestibulum justo imperdiet, wisi risus purus augue vulputate
+                voluptate neque, curabitur dolor libero sodales vitae elit massa.Lorem ipsum dolor sit amet, maecenas
+                eget vestibulum justo imperdiet, wisi risus purus
+                augue vulputate voluptate neque, curabitur dolor libero sodales vitae elit massa.
+            </p>
+            <p>
+                Lorem ipsum dolor sit amet, maecenas eget vestibulum justo imperdiet, wisi risus purus augue vulputate
+                voluptate neque, curabitur dolor libero sodales vitae elit massa.
+            </p>
+        </div>
+        <div class="about__video">
+            <iframe src="https://www.youtube.com/embed/2ZKV6B03I0s" frameborder="0"
+                allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
```

Comment:
I my research, I found that libraries, even bootstrap, use iframe for display youtube videos. There is a library that imitates the video tag, but in this case I prefer leave this way, because here I am not needing the features that video tag provides.